addons/sys_core/stringtable.xml

```diff
@@ -376,6 +388,10 @@
             <Portuguese>Ignorar Direção da Antena</Portuguese>
             <Italian>Ignora direzione antenna</Italian>
         </Key>
+        <Key ID="STR_ACRE_sys_core_autoAntennaDirection_displayName">
+            <English>Automatically Adjust Antenna Direction</English>
```

Comment:
Slightly long, but I have no idea how to shorten. Maybe `Automatic Antenna Direction`?